README.md

```diff
@@ -55,9 +55,46 @@ All variables which can be overridden are stored in [defaults/main.yml](defaults
 ### Playbook
 
 ```yaml
+---
 - hosts: all
+  any_errors_fatal: true
```

Comment:
Not everyone need this and there is no reliance on this option in a role. Please remove.